### DISCUSSION TITLE
how to autobreak page

### DISCUSSION THREAD

**Discussion by dualer** [marp-team/marp#245] [Q&A] [answered]
Like LaTeX beamer's `[allowframebreaks]`, when contents in one page is beyond scope, can  `marp` autobreak the page and put extra part into a new page?

**Comment by yhatt** [comment, marked as answer]
Cannot. Marp is based on web technology and HTML allows various features that the element size will be unpredictable, like a remote image, CSS animation, and CSS transitions. Marp cannot know the actual height of slide page before output HTML, so auto-break also cannot.
